.fixtures.yml

```diff
@@ -120,5 +120,7 @@ fixtures:
     upstart: https://github.com/simp/pupmod-simp-upstart
     useradd: https://github.com/simp/pupmod-simp-useradd
     xinetd: https://github.com/simp/pupmod-simp-xinetd
+    '../acceptance/rsync_shares/simp':
```

Comment:
This couples the tests to a module that may be changing.  Do you really want to do that?  At a minimum, if you choose to do this, you should download a tagged version, so the content is assured.